README.md

```diff
@@ -80,15 +79,107 @@ Depending on the scenario a DomainError will be returned stating the problem. If
 A payment will time out within 35 minutes, meaning that the whole process of user accepting, callbacks made and authorization must be completed within 35 minutes.
 Furthermore after you get the callback containing the card data, you must update the status of the authorization to either "authorize-succesfull" or "authorize-failed" within 32 seconds to ensure a smooth experience for the user waiting for the confirmation.
 
-## Dankort acquiring
+## Strong Customer Authentication (SCA)
+
+We aim to ensure Delegated Authentication (DA). This means that responsibility for authenticating the customer/payer no longer lies with the Issuer, but is delegated to MobilePay.  When/if we fail, and the Issuer is responding to an authorisation attempt with a Soft Decline/"step-up", a 3-D Secure fallback solution must be in place.
+
+### Delegated Authentication for Dankort
+As long as you use the tags and values described here, all is well. Nets will recognize MobilePay and trust our authentication process.
+
 Use POS code: ‘K005K0K00130’.
-### Using Nets SDI secification
+#### Using Nets SDI specification
 In Field S120 tag 36: the value of 8844101001</br>
 In Field S120 tag 70 pos 14 (exemption Tag): the value of 3 for Delegated Authentication
-### Using Nets TRG PSIP/ ISO 8583 / Merchant Guide SSL
+#### Using Nets TRG PSIP/ ISO 8583 / Merchant Guide SSL
 In Field 47 tag 7R: the value of 8844101001</br>
 In Field 47 tag V!: the value of 23
 
+### Delegated Authentication for all Visa Cards
+When you initiate a payment, make sure to use v2 of the API. Here you give a tokenCallbackUrl for all accepted Visa types. However, please also provide a carddataCallbackUrl as failover, because not all Visa cards can be tokenized. For Dynamic Linking, please give us MerchantUrl and MerchantName, exactly as registred by the merchant with the Acquirer.</br>
```

Comment:
> `MerchantUrl` and `MerchantName`, exactly as registred by the merchant with the Acquirer.

The merchant's URL is not (necessarily) part of the authorization message. Is it perhaps supposed to mean the "descriptor" otherwise known as the "text on statement"?

Reply:
As I understand it, both MC and Visa is asking for it. However, I learned just now, by talking to MC that they´ll compare data from us like this:
1. When they recieve the "payment request" / asked to issue a cryptogram, they set the "base". 
2. When they get the cryptogram back (from Issuer) for verification, they´ll compare what they nowrecieved in the Crypto, to what they´ve stored as "base" when they issued the Crypto. 
So, basically, in a happy day, those will always match nicely.
I´m deleting the part about "exactly as registrered by the Merchant with the Acquirer". But I think we´re best served by keeping it as "MerchantName" and "MerchantUrl", not to cause unnessary confusion.